src/main/webapp/i18n/en/global.json

```diff
@@ -56,12 +56,7 @@
             "info": {
                 "authenticated": {
                     "prefix": "Are you a registered user? ",
-                    "link": "Sign in",
-                    "suffix": ", you can try the default accounts:<br/>- Administrator (login=\"admin\" and password=\"admin\") <br/>- User (login=\"user\" and password=\"user\")."
-                },
-                "register": {
-                    "noaccount": "You don't have an account yet?",
-                    "link": "Register a new account"
+                    "link": "Sign in"
```

Comment:
Can you please change the link in the `footer` with the correct link https://radar-base.atlassian.net/wiki/spaces/RAD/pages/1913978881/User+Guide+for+Management+Portal,

Reply:
Done.

Reply:
Thank you :+1: